apps/meteor/tests/e2e/omnichannel/omnichannel-livechat-api.spec.ts

```diff
@@ -66,6 +66,13 @@ const createFakeVisitorRegistration = (extra?: { department?: string }) => ({
 
 test.describe('OC - Livechat API', () => {
 	// TODO: Check if there is a way to add livechat to the global window object
+	test.beforeAll(async ({ api }) => {
+		expect((await api.post('/settings/Livechat_Require_Contact_Verification', { value: 'never' })).status()).toBe(200);
+	});
+
+	test.afterAll(async ({ api }) => {
+		expect((await api.post('/settings/Livechat_Require_Contact_Verification', { value: 'always' })).status()).toBe(200);
```

Comment:
dont check status inside after blocks, if fails prevent the next ones (ok here its single)